docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
-                           record_from_solution = (x, p) -> x[plot_idx], J = J)
-nothing   # hide
-```
-Next, we need to specify the input options for the pseudo-arclength continuation
-method (PACM) which produces the diagram.
+The previous bifurcation diagram can be computed, with these various options specified, in the following way:
 ```@example ex1
-bopts = ContinuationPar(dsmax = 0.05,          # Max arclength in PACM.
-                        dsmin = 1e-4,          # Min arclength in PACM.
-                        ds = 0.001,            # Initial (positive) arclength in PACM.
-                        max_steps = 100000,     # Max number of steps.
-                        p_min = p_span[1],      # Min p-val (if hit, the method stops).
-                        p_max = p_span[2],      # Max p-val (if hit, the method stops).
-                        detect_bifurcation = 3) # Value in {0,1,2,3}
-nothing   # hide
+p_span = (2.0, 20.0)
+opt_newton = NewtonPar(tol = 1e-9, max_iterations = 1000)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], 
+                          dsmin = 0.001, dsmax = 0.01, max_steps = 1000,
+                          newton_options = opt_newton)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+nothing # hide
 ```
-Here `detectBifurcation` determines to what extent bifurcation points are
-detected and how accurately their values are determined. Three indicates to use the most
-accurate method for calculating their values.
+(however, in this case these additional settings have no significant effect on the result)
 
-We are now ready to compute the bifurcation diagram:
+## Bifurcation diagrams with disjoint branches
+Let's consider the previous case, but instead compute the bifurcation diagram over the interval *(2.0, 15.0)*:
 ```@example ex1
-bf = bifurcationdiagram(bprob, PALC(), 2, (args...) -> bopts)
-nothing   # hide
+p_span = (2.0, 15.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps = 1000)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+plot(bif_dia; xguide="k1", yguide="X")
 ```
-Finally, we can plot it:
-```@example ex1
-plot(bf; xlabel = string(bif_par), ylabel = string(plot_var))
+Here, in the bistable region, we only see a single branch. The reason is that the continuation algorithm start at our initial guess (here made at *k1 = 4.0* for *(X,Y) = (5.0,2.0)*) and tracks the diagram from there. However, with the upper bound set at *k1=15.0* the bifurcation diagram have a disjoint branch structure, preventing the full diagram from being computed by continuation alone. In this case it could be solved by increasing the bound of *k1=15.0*, however, this is not possible in all cases. In these cases, *deflation* can be used. This is described in the [BifurcationKit documentation](https://bifurcationkit.github.io/BifurcationKitDocs.jl/dev/tutorials/tutorials2/#Snaking-computed-with-deflation).
```

Comment:
```suggestion
Here, in the bistable region, we only see a single branch. The reason is that the continuation algorithm starts at our initial guess (here made at `k1 = 4.0` for `(X,Y) = (5.0,2.0)`) and tracks the diagram from there. However, with the upper bound set at `k1=15.0` the bifurcation diagram has a disjoint branch structure, preventing the full diagram from being computed by continuation alone. In this case it could be solved by increasing the bound from `k1=15.0`, however, this is not possible in all cases. In these cases, *deflation* can be used. This is described in the [BifurcationKit documentation](https://bifurcationkit.github.io/BifurcationKitDocs.jl/dev/tutorials/tutorials2/#Snaking-computed-with-deflation).
```
Please don't use italic blocks (i.e. stars), either use code or math blocks as appropriate. I'm going to stop commenting on this, but please take a pass and update the tutorial to use one of these two.

Reply:
will do
(as mentioned this was before you told me about the $$, and I failed to go back and look at the old tutorials, I agree ** shoudln't be used in these cases as there are better alts)